OneSignalSDK/onesignal/src/main/java/com/onesignal/GenerateNotification.java

```diff
@@ -240,28 +240,9 @@ private static void setAlertnessOptions(JSONObject fcmJson, NotificationCompat.B
          } catch (Throwable t) {
             notificationDefaults |= Notification.DEFAULT_LIGHTS;
          } // Can throw if an old android support lib is used or parse error.
-      }
-      else
+      } else {
          notificationDefaults |= Notification.DEFAULT_LIGHTS;
-
-      if (OneSignal.getVibrate() && fcmJson.optInt("vib", 1) == 1) {
```

Comment:
We just want to remove the `OneSignal.getVibrate()` call here. As a backwards compatibility for pre-O devices the OneSignal backend will still send "vib" depending if the Notification Category has this set.